account_chatter/__manifest__.py

```diff
@@ -1,9 +1,6 @@
-# Copyright 2020 Vauxoo
-# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
```

Comment:
Apply this change in v15.

BTW, I noticed they were not removed from file `models/__init__.py`, along with all non-migration changes.